incompatibility/utils/utils.go

```diff
@@ -0,0 +1,30 @@
+package utils
+
+import (
+	"os/exec"
+
+	openapiv3 "github.com/googleapis/gnostic/openapiv3"
+	"google.golang.org/protobuf/proto"
+)
+
+func ParseOpenAPIDoc(input string) (*openapiv3.Document, error) {
```

Comment:
since this is used for testing I think you can mark this with _test.go to ensure it's only built during the testing phase.

Reply:
Is there a way to avoid the duplicated code? Both functions are used in the generator. Could we define a utils file in the generator and import the functions in the incompatibility package?

Reply:
@LorenzHW  yes i added a utils file for ParseOpenAPIDoc and CreateOpenAPIDocFromGnosticOutput and updated the code in generator/ to import these functions from this file